types/structured-data.d.ts

```diff
@@ -7,6 +7,21 @@
 declare global {
   module LH {
     module StructuredData {
+
+      export type ValidatorType =
+        | "json"
+        | "json-ld"
+        | "json-ld-expand"
+        | "schema-org";
```

Comment:
ok, where is this leading `|` coming from :P Not even prettier does this, I don't think

Reply:
my prettier definitely does this in other projects, not sure about here :)

Reply:
> my prettier definitely does this in other projects, not sure about here :)

maybe it's a typescript prettier thing? I [couldn't get it to repro in their playground ](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA3AhgJwAQHcAW68qcOAvNgIzYA+2ATLdgMxMAsTArEwGxMDsTABxMAnE0oAGCdTqVGc1nI5zucvnMFyRc8XXqSA3CAA0ICAAcYAS2gBnZKCyYIuAApYEDlOlQRrACamIABGmOhgANZwMADKFhHWUADmyDCYAK5wZvgwALYANgDq+NbwdglgcLFeZdaoZQCeyOB2DmZJdqQwbuHJeejIAGboBV1mAFZ2AB4AQuFRMbHoeXAAMklww6PjIFPTsUnJBXAAihkQ8Ntj2SAJmF2YLSHoIY0F0MEWmEkwRYEwfDIISSMzfCBdIrhCwtb5wR4kYIARwu8F6lm8IHQdgAtFA4HAAoTgpg4CjrKTeuh+oMkCMbmYunlrGlMrc7EcTudLls6TtbjBXv8AoDkPQzOl0NYCkcAMIQPIDFpQaBbMwZLoAFVe3npXQAvvqgA)